desktop/src/config.rs

```diff
@@ -0,0 +1,32 @@
+use crate::cli::Opt;
```

Comment:
Maybe it's just me, but having `GlobalPreferences` in `config.rs` feels weird, do we need to mixup these synonyms?

Reply:
That's fair, I can rename it to "preferences"

I started calling it "config" initially but "preferences" felt more appropriate for global ruffle-specific stuff, and "config" maybe for the per-swf stuff later